eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -17,35 +17,42 @@ As a front-end developer I would like to:
 * Manage the content via the REST API:
     * Use the REST endpoints to quickly create a new object
     * Use the REST endpoints to quickly update an already created object
-    * Use the REST endpoint
 
 # Creating content POST
 The create content endpoint should be simplified to default most values and the simplest possible.
 
 This call should create:
 * A new content object (not a draft) and publish
 * Assign ownership to the current user
-* Create the object in the default language
+* Create the object in the default language if not set
 * Location can be defined either with ID or URL
 
 ## Example: creating a new object
 
-```
-Resource: /content/object
-Method: POST
+__Resource__: _/content/object_
+
+__Accept__: _application/vnd.ez.api.Content+json_
+
+__Content-Type__: _application/vnd.ez.api.SimpleContentCreate+json_
+
+__Method__: _POST_
+
+__Body__:
+```json
 {
-    "content-location": "/blogs",
-	"content-type": "blog_post",
+    "ParentLocation": <ID> or <URL>,
```

Comment:
I was actually not against this one... While it is convenient that you can pass all the options for creating the location, having a very simple shortcut to tell "this is where it goes" is I think useful.

Maybe the part that should be hidden is the "LocationCreate". A lambda API user may not really care that publishing a new content requires that a new location is created. All we care about is the parent location.